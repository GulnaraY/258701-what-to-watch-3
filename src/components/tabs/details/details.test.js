import React from 'react';
import rerender from 'react-test-renderer';
import Details from './details.jsx';

const filmDetails = {
  genre: `drame`,
  release: 1999,
  director: `Tim Cook`,
  actors: [`actress`, `actor`],
  runTime: `1h 30m`,
};

it(`Should render Details`, () => {
  const {genre, release, runTime, director, actors} = filmDetails;
  const tree = rerender.create(
      <Details
        director={director}
        actors={actors}
        genre={genre}
        release={release}
        runTime={runTime}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
