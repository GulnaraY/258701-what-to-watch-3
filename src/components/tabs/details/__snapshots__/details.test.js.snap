// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Details 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Tim Cook
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        actress, actor
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        10
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        drame
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        1999
      </span>
    </p>
  </div>
</div>
`;
